.travis.yml

```diff
@@ -26,23 +26,18 @@ jobs:
     - stage: docker
   include:
     - stage: lint
-      python: 3.6
+      python: 3.8
       env: TOXENV=bandit
-    - python: 3.6
+    - python: 3.8
       env: TOXENV=format
-    - python: 3.6
+    - python: 3.8
       env: TOXENV=lint
       script:
         - bash -c 'shopt -s globstar nullglob; shellcheck **/*.sh'
         - tox
-    - python: 3.6
+    - python: 3.8
       env: TOXENV=docs
 
-    - stage: test
-      python: 3.6
-      env: TOXENV=py36
-    - python: 3.7
```

Comment:
Why removing python 3.7?

Reply:
Could you add mypy to .travis.yml?

Reply:
Because https://www.python.org/dev/peps/pep-0589/ - Typed Dict has been introduced in 3.8

Reply:
Ok, could you reflect this change in setup.py, it's used for creating metadata for PyPi packages about supported python versions.